Sadly not that easy:

[46](https://github.com/ESUG/esug.github.io/actions/runs/14708768938/job/41275493511?pr=382#step:2:47)
E: Unable to locate package libssl1.1:i386
[47](https://github.com/ESUG/esug.github.io/actions/runs/14708768938/job/41275493511?pr=382#step:2:48)
E: Couldn't find any package by glob 'libssl1.1'
[48](https://github.com/ESUG/esug.github.io/actions/runs/14708768938/job/41275493511?pr=382#step:2:49)
E: Couldn't find any package by regex 'libssl1.1'
[49](https://github.com/ESUG/esug.github.io/actions/runs/14708768938/job/41275493511?pr=382#step:2:50)
Error: Process completed with exit code 100.

--- a/.github/workflows/makefile.yml
+++ b/.github/workflows/makefile.yml
@@ -17,8 +17,7 @@ jobs:
       run: |
         sudo dpkg --add-architecture i386
         sudo apt-get update
-        sudo apt-get install -y libasound2:i386 libssl1.1:i386 libfreetype6:i386
-
+        
     - uses: actions/checkout@v4
       
     - name: Generate

--- a/.github/workflows/makefile.yml
+++ b/.github/workflows/makefile.yml
@@ -16,8 +16,9 @@ jobs:
     - name: Install dependencies
       run: |
         sudo dpkg --add-architecture i386
-        sudo apt-get install -y libasound2:i386 libfreetype6:i386	
         sudo apt-get update
+        sudo apt-get install -y libasound2:i386 libfreetype6:i386	
+
         
     - uses: actions/checkout@v4
       

--- a/.github/workflows/makefile.yml
+++ b/.github/workflows/makefile.yml
@@ -16,6 +16,7 @@ jobs:
     - name: Install dependencies
       run: |
         sudo dpkg --add-architecture i386
+        sudo apt-get install -y libasound2:i386 libfreetype6:i386	
         sudo apt-get update
         
     - uses: actions/checkout@v4

